package.json

```diff
@@ -1,23 +1,20 @@
 {
   "private": true,
   "workspaces": [
-    "packages/*"
+    "packages/**"
   ],
   "scripts": {
-    "build": "turbo run build",
-    "dev": "turbo run dev --parallel",
     "lint": "turbo run lint",
     "test": "turbo run test",
-    "clean": "turbo run clean && rm -rf node_modules",
-    "format": "prettier --write \"**/*.{js,md}\""
+    "prettier": "turbo run prettier",
+    "prepare": "husky install"
   },
   "devDependencies": {
-    "prettier": "^2.5.1",
```

Comment:
Why was prettier removed?

Reply:
It is not called from root anymore; each package has its own lint/format scripts

Reply:
I don't know if it's better to have Prettier in each package. I get that it's better for Turborepo, but Prettier can also format files in the root of the repo.

Reply:
I'd be content moving prettier to the root and have it just format _everything_. This is only getting fired off during the precommit